config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits-main.yaml

```diff
@@ -275,7 +275,7 @@ presubmits:
     spec:
       serviceAccountName: prowjob-default-sa
       containers:
-      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20220829-d5650c79bf-1.24
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20220829-d5650c79bf-1.25
         args:
           - runner.sh
           - "./scripts/ci-e2e.sh"
```

Comment:
There is one more image below (pull-cluster-api-e2e-workload-upgrade-1-21-1-22-main)

Reply:
@oscr Can you please also open a second PR to do the same in cluster-api-periodics-main-upgrades.yaml and cluster-api-periodics-main.yaml
(intentionally a separate PR so I can merge the second one after I know the presubmit jobs are working after the first one)

Reply:
@sbueringer Great catch! I don't know how I missed it, and you where able to catch it. This pr is updated. I will fix a separate as requested.

Reply:
Perfect, thx! :)